torchmultimodal/modules/losses/flava.py

```diff
@@ -438,7 +442,7 @@ def forward(
                 else -(text_masked_sequence.size(1) - 1)
             )
             sequence_for_text = multimodal_masked_sequence[:, start_index:, :]
-            outputs.mmm_text_output = self.mmm_loss.mlm(
+            outputs.mmm_text_output = self.mmm_loss["mlm"](
```

Comment:
why?

Reply:
When I use dot access mypy thinks it's a tensor

Reply:
just put ignore for now and add that comment